src/custom-element.js

```diff
@@ -1434,10 +1447,16 @@ function createBaseAmpElementProto(win) {
  */
 export function registerElement(win, name, implementationClass) {
   knownElements[name] = implementationClass;
+  const klass = createBaseCustomElementClass(win, name);
 
-  win.document.registerElement(name, {
-    prototype: createAmpElementProto(win, name),
-  });
+  const supportsCustomElementsV1 = 'customElements' in win;
+  if (supportsCustomElementsV1) {
+    win['customElements'].define(name, klass);
```

Comment:
we could add it in our current extern, more correct thing to do is open an issue internally or in the lib to add the new custom element v1 externs if their not there yet